picard/tagger.py

```diff
@@ -480,6 +485,16 @@ def handle_command_fingerprint(self, argstring):
         for album_name in self.albums:
             self.analyze(self.albums[album_name].iterfiles())
 
+    def handle_command_load(self, argstring):
+        if argstring.startswith("command://"):
+            log.error("Cannot LOAD a command: %s", argstring)
+            return
+
+        if not argstring.startswith("file://") and not argstring.startswith("mbid://") and not argstring.startswith("url://"):
+            argstring = "file://" + argstring
```

Comment:
This prevents something like `-e load https://musicbrainz.org/release/f1e04a39-65fc-48c9-b9f9-fa1dbe7e9326`.
Instead a check like the one at https://github.com/metabrainz/picard/blob/974547ce32718b35d8da85445887856fa4062fb5/picard/tagger.py#L1335 could be used.

Reply:
My bad, seems like I've gone too automatic... Tho I'd rather keep it consistent with our `ParseItemsToLoad` class, see yourself

Reply:
Actually, none of it is needed. See the recent commit.